core/src/test/scala/org/apache/spark/executor/CoarseGrainedExecutorBackendSuite.scala

```diff
@@ -224,6 +223,30 @@ class CoarseGrainedExecutorBackendSuite extends SparkFunSuite
     }
   }
 
+  test("number of retries is respected when always failing") {
+
```

Comment:
remove space here and bellow, pls follow the format the other tests have.

Reply:
Hi @skonto 

This class has a Main method and reads the values from the argv. In my opinon is an overkill to add it to define the numbers of the retries. Let me know if what you want is to me add arguments to the main method or you prefer me to leave it as it is.

Reply:
I just don't think this test or method is worthwhile. It's more or less verifying you've simulated a for loop.

Reply:
It tests mainly two things: 1) That you indeed go n times, and 2) the behaviour of throwing the exception.   I can change it if you want, for sure there is not performance difference and I think it will be actually less lines of code than the try catch and the if statements. 
@srowen let me know if you want me to go to the iterative approach.

Reply:
I would keep it simple and just write a for loop.

Reply:
Test will be deleted then.